extractorapp/src/main/java/org/georchestra/extractorapp/ws/extractor/task/ExtractionTask.java

```diff
@@ -428,9 +428,9 @@ private void statSetRunning() {
 					"VALUES (?, ?, ?, ?)",
 					Statement.RETURN_GENERATED_KEYS);
 
-			pst.setString(1, this.requestConfig.username);
-			pst.setArray(2, c.createArrayOf("varchar", this.requestConfig.roles.split("\\s*;\\s*")));
-			pst.setString(3, this.requestConfig.org);
+			pst.setString(1, this.requestConfig.username == null ? "" : this.requestConfig.username);
```

Comment:
I would have prefered to set these fields to NULL, eventually making an evolution to the schema.
https://github.com/georchestra/georchestra/blob/16.12/postgresql/09-extractor-app.sql